### PR TITLE
Blogging Prompts List: initial view

### PR DESCRIPTION
**Pull request by ScoutHarris**
Ref: #18502

This adds an initial Blogging Prompts list view.

To test:
- Enable `bloggingPrompts` feature flag.
- Select the My Site `Home` tab.
- On the prompt card menu, select `View more prompts`.
- Verify the `Prompts` view is displayed.
  - 10 stub empty rows.
  -  Filter tab bar at the top under the nav bar.

https://user-images.githubusercontent.com/1816888/166591274-2bacf16c-617b-4e48-b164-57755f07fde2.mp4

## Regression Notes
1. Potential unintended areas of impact
N/A

2. What I did to test those areas of impact (or what existing automated tests I relied on)
N/A

3. What automated tests I added (or what prevented me from doing so)
N/A

PR submission checklist:

- [x] I have completed the Regression Notes.
- [x] I have considered adding unit tests for my changes.
- [x] I have considered adding accessibility improvements for my changes.
- [x] I have considered if this change warrants user-facing release notes and have added them to `RELEASE-NOTES.txt` if necessary.

### CODE REVIEW

File: WordPress/Classes/ViewRelated/Blog/Blogging Prompts/BloggingPromptsViewController.swift [hunk 1]
```diff
@@ -0,0 +1,101 @@
...
+        WPStyleGuide.configureFilterTabBar(filterTabBar)
+        filterTabBar.items = PromptFilter.allCases
+        filterTabBar.addTarget(self, action: #selector(selectedFilterDidChange(_:)), for: .valueChanged)
+    }
+
+    @objc func selectedFilterDidChange(_ filterTabBar: FilterTabBar) {
+        // TODO:
+        // - track selected filter changed
+        // - refresh view for selected filter
+        }
```
Comment (wargcm) [inline WordPress/Classes/ViewRelated/Blog/Blogging Prompts/BloggingPromptsViewController.swift]: [Nitpick]: Indentation is off for this closing brace.

File: WordPress/WordPress.xcodeproj/project.pbxproj [hunk 2]
```diff
@@ -12737,19 +12752,20 @@
...
 				3F43603623F36962001DEE70 /* Blog Selector */,
-				3F37609B23FD803300F0D87F /* Blog + Me */,
+				98A047702821BFB6001B4E2D /* Blogging Prompts */,
+				3F170D2A2654615900F6F670 /* Blogging Reminders */,
+				1716AEFA25F2926C00CF49EC /* My Site */,
+				C77FC90628009C3C00726F00 /* Onboarding Questions Prompt */,
 				4395A15E210672C900844E8E /* QuickStart */,
 				E6431DDE1C4E890B00FD8D90 /* Sharing */,
 				FA5C74091C596E69000B528C /* Site Management */,
+				FA73D7E72798766300DF24B3 /* Site Picker */,
```
Comment (wargcm) [inline WordPress/WordPress.xcodeproj/project.pbxproj]: Nice! 💯

**Review by wargcm — APPROVED** [review]
Looks good!

File: WordPress/Classes/ViewRelated/Blog/Blogging Prompts/BloggingPromptsViewController.swift (hunk 1 above)
Comment (ScoutHarris) [reply WordPress/Classes/ViewRelated/Blog/Blogging Prompts/BloggingPromptsViewController.swift]: Woops. Thanks!

**Comment by ScoutHarris** [comment]
Thanks @wargcm !

**Pull request merged**